src/components/App.tsx

```diff
@@ -93,37 +117,48 @@ const App: FC<AppProps> = ({ client }) => {
     );
   };
 
-  return (
-    <FHIRClientProvider client={client}>
-      <PatientProvider>
-        <PathwayProvider
-          pathwayCtx={{
-            evaluatedPathway: currentPathway,
-            setEvaluatedPathway: setEvaluatedPathwayCallback,
-            updateEvaluatedPathways: updateEvaluatedPathways
-          }}
-        >
-          <div>
-            <Header logo={logo} />
-            <Navigation
-              evaluatedPathways={evaluatedPathways}
-              selectPathway={selectPathway}
-              setSelectPathway={setSelectPathway}
-            />
-          </div>
-          {selectPathway ? (
-            <PathwaysList
-              evaluatedPathways={evaluatedPathways}
-              callback={setEvaluatedPathwayCallback}
-              service={service}
-              resources={patientRecords}
-            ></PathwaysList>
-          ) : (
-            <PatientView evaluatedPathway={currentPathway} />
-          )}
-        </PathwayProvider>
-      </PatientProvider>
+  const renderPathwayProvider = (): ReactNode => {
```

Comment:
This one is a little weird because I'm not actually creating a new component.  I just wanted to render that component with the same exact children and props or else there would be a lot of duplication on lines 154 and 160.  Do you think I should still make a FunctionalComponent?

Reply:
I think its still worth making it a FunctionalComponent for the JSX syntax alone.  That way we can consistently use the JSX syntax and it is immediately clear to anyone that looks at it that a React Element is returned.  I rarely see the JSX that returns components with a function call like we do here (e.g. `foo()` vs `<Foo/>`).

Another way to think about it could be as a container vs presentational component.  This is still a component, but just a presentational subcomponent as we break everything down.

Reply:
That makes sense.  Should be addressed now.